### DISCUSSION TITLE
Using draft releases for nightly builds makes GH release notifications pointless

### DISCUSSION THREAD

**Discussion by andyreidz** [keycloak/keycloak#10914] [Feedback]
I personally depend on GH notifications of new releases, having a notification for every nightly draft release generates a lot of noise.

**Comment by stianst** [comment]
I appreciate the noise it generates, but don't think there's a possible solution on our end to this problem as we need to leverage GitHub releases for nightly builds.

**Reply by stianst** [reply]
Made some adjustments and are now deleting artifacts from the nightly release and uploading new ones instead of deleting the release and creating a new release. Hopefully that works, if not there's nothing I can think of

**Reply by andyreidz** [reply]
Thank you for this change, I'll see if there are anymore release notifications over the weekend.
Have a great one!
